mod/racing_plus/Race.lua

```diff
@@ -115,6 +132,90 @@ function Race:PostUpdateCheckKeeperHolyMantle()
   end
 end
 
+function ModServer:Connect()
```

Comment:
probably put ModServer in a separate file, since you are using a separate object?